thirdeye/thirdeye-frontend/app/pods/rootcause-scores-cache/service.js

```diff
@@ -0,0 +1,115 @@
+import Ember from 'ember';
+import { trimTimeRanges, filterPrefix, toBaselineRange } from 'thirdeye-frontend/utils/rca-utils';
+import { checkStatus } from 'thirdeye-frontend/utils/utils';
+import fetch from 'fetch';
+import _ from 'lodash';
+
+export default Ember.Service.extend({
```

Comment:
Let's use he newest module api for new files please. https://github.com/emberjs/rfcs/blob/master/text/0176-javascript-module-api.md#addendum-1---table-of-module-names-and-exports-by-global

Reply:
Ok. I'll update the services code in a single followup PR rather than have a mixed state